frontend/cypress/pages/NonConformPage.js

```diff
@@ -3,7 +3,7 @@ class NonConform {
     return cy.get("h2");
   }
 
-  getViewNonConformTitle(){
+  getViewNonConformTitle() {
```

Comment:
Are these changes part of the issue youre working on ??

Reply:
No sir,

Reply:
How else can I resolve this

Reply:
MR @mozzy11 if i happen to ask for , for narration on the way to remove the unwanted change i got it a little